icons/stackoverflow/stackoverflow-plain-wordmark.svg

```diff
@@ -0,0 +1 @@
4 1.098-1.086-1.157-1.238-2.137-1.645-3.68-1.645-2.84 0-4.988 1.93-4.988 5.786 0 3.855 2.148 5.785 4.988 5.785 1.555 0 2.539-.414 3.707-1.676l-1.121-1.082c-.848.941-1.512 1.285-2.586 1.285a3.042 3.042 0 0 1-2.61-1.285c-.527-.734-.73-1.606-.73-3.027 0-1.426.203-2.297.73-3.032a3.042 3.042 0 0 1 2.61-1.285Zm14.11-1.348h-2.055l-5.106 4.985.008-10.02h-1.648v16.344h1.648l-.008-4.121 2-2.012 3.774 6.133h2.039l-4.672-7.277Zm7.609-.277c-1.688 0-2.828.66-3.528 1.406-1.03 1.079-1.297 2.375-1.297 4.446 0 2.09.266 3.386 1.297 4.465.7.746 1.84 1.406 3.528 1.406 1.691 0 2.851-.66 3.554-1.406 1.032-1.079 1.293-2.375 1.293-4.465 0-2.07-.261-3.368-1.293-4.446-.703-.746-1.863-1.406-3.554-1.406Zm1.34 8.645c-.36.347-.844.527-1.34.504a1.735 1.735 0 0 1-1.313-.504c-.594-.594-.66-1.606-.66-2.793 0-1.188.066-2.18.66-2.774a1.75 1.75 0 0 1 1.313-.504c.496-.023.98.157 1.34.504.59.594.656 1.586.656 2.774 0 1.187-.067 2.199-.656 2.793Zm11.097-8.516-2.3 7.086-2.325-7.086H57.82l4.211 11.461h2.235l4.187-11.461Zm7.856-.129c-2.961 0-4.977 2.11-4.977 5.852 0 4.64 2.586 5.87 5.285 5.87 2.059 0 3.18-.636 4.34-1.804l-1.73-1.691c-.727.726-1.34 1.078-2.59 1.078-1.602 0-2.5-1.078-2.5-2.555h7.148v-1.273c0-3.145-1.797-5.477-4.976-5.477Zm-2.172 4.793a3.11 3.11 0 0 1 .265-1.23 2.128 2.128 0 0 1 1.907-1.188c.808 0 1.547.461 1.906 1.188.172.386.262.808.266 1.23Zm11.355-3.562v-1.102h-2.785v11.461h2.852v-6.906c0-1.453.965-2.114 1.84-2.114.578-.02 1.132.23 1.503.676l2.16-2.168c-.788-.793-1.578-1.078-2.699-1.078a3.852 3.852 0 0 0-2.87 1.23Zm6.453-2.297v12.644h2.852v-9.035h2.105v-2.176h-2.105v-1.254c0-.66.328-1.035 1.008-1.035h1.097v-2.422h-1.625c-2.324 0-3.332 1.653-3.332 3.278Zm16.692 1.062c-1.688 0-2.828.66-3.531 1.41-1.028 1.079-1.293 2.375-1.293 4.442 0 2.09.265 3.386 1.293 4.469.703.746 1.843 1.406 3.53 1.406 1.688 0 2.852-.66 3.555-1.406 1.028-1.079 1.293-2.38 1.293-4.47 0-2.066-.265-3.362-1.293-4.44-.703-.75-1.867-1.41-3.554-1.41Zm1.34 8.649c-.36.343-.844.527-1.34.504a1.764 1.764 0 0 1-1.317-.504c-.59-.598-.656-1.61-.656-2.797 0-1.188.067-2.176.656-2.77a1.752 1.752 0 0 1 1.317-.508c.496-.023.98.16 1.34.508.59.594.656 1.582.656 2.77 0 1.187-.067 2.199-.656 2.797Zm16.976-8.516-1.844 7.086-2.347-7.086h-2.082l-2.324 7.086-1.84-7.086h-3.028l3.508 11.461h2.348l2.39-7.191 2.391 7.191h2.344l3.488-11.461Zm-25.847 8.008v-12.48h-2.848v12.656c0 1.629.984 3.277 3.332 3.277h1.621v-2.418h-1.094c-.722 0-1.008-.352-1.008-1.035Zm-83.973-6.067 1.523-1.527h-3.117v-3.719h-1.648v11.73c0 1.677.96 3.032 2.883 3.032h1.168v-1.422h-.872c-1.05 0-1.53-.62-1.53-1.652v-6.442Zm0 0"/><path style="fill-rule:evenodd;fill:#bbb;fill-opacity:1" d="M88.754 61.574h8.414v33.867H21.422V61.574h8.418v25.403h58.914Zm0 0"/><path style="fill-rule:nonzero;fill:#f58025;fill-opacity:1" d="m39.012 59.059 41.336 8.738 1.738-8.313-41.336-8.742Zm5.468-19.91 38.293 17.937 3.567-7.703-38.293-17.938ZM55.078 20.25 87.54 47.441l5.406-6.527-32.46-27.195ZM76.031.148l-6.781 5.07 25.21 34.098 6.778-5.07Zm-37.773 78.36h42.078v-8.465H38.258Zm0 0"/></svg>
```

Comment:
The paths need to be unified here since it's a plain icon. So you should end up with just one `<path ... />` element